client/src/components/MoreHomes.jsx

```diff
@@ -7,11 +7,21 @@ class MoreHomes extends React.Component {
     super(props);
 
     this.state = {
+      height: window.innerHeight,
```

Comment:
It's pretty cool to use `window.innerHeight` and `window.innerWidth` to handle window resizing. 
Will this function be affected while we combine everything else together? (ex. combine with "reviews" , "photo gallery", "room info" , and "reservation")
I am just curious since I am not familiar with these 2 variables. :)

Reply:
For now I would guess that it will not affect. We will see next week..